src/components/TransactionPreview/ActionPreview.tsx

```diff
@@ -818,6 +858,184 @@ const ActionPreview = ({
+                  />
+                  <div>
+                    <Text size={16} marginBottom={1} medium block>
+                      {fromAmount} {fromAsset.symbol}
+                    </Text>
+                    <Text size={12}>On {fromChainTitle}</Text>
+                  </div>
+                </ValueWrapper>
+              </TransactionAction>
+              <TransactionAction>
+                <ColoredText>You receive</ColoredText>
+                <ValueWrapper>
+                  <CombinedRoundedImages
+                    title={toAsset.symbol}
+                    url={toAsset.iconUrl}
+                    smallImageTitle={toChainTitle}
+                    smallImageUrl={toNetwork?.iconUrl}
+                  />
+                  <div>
+                    <Text size={16} marginBottom={3} medium block>
+                      {toAmount} {toAsset.symbol}
+                    </Text>
+                    <Text size={12}>On {toChainTitle}</Text>
+                  </div>
+                </ValueWrapper>
+              </TransactionAction>
+            </DoubleTransactionActionsInSingleRow>
+            {!!senderAddress && !!receiverAddress && (
+              <TransactionAction>
+                <Text size={16} medium>
+                  <>
+                    From &nbsp;
+                    <ClickableText onClick={() => onCopy(senderAddress)}>
+                      {humanizeHexString(senderAddress)}
+                    </ClickableText>
+                    &nbsp;
+                  </>
+                  to &nbsp;
+                  <ClickableText onClick={() => onCopy(receiverAddress)}>
+                    {humanizeHexString(receiverAddress)}
+                  </ClickableText>
+                </Text>
+              </TransactionAction>
+            )}
+          </>
+        ) : (
+          <>
+            {previewList.map((preview) => {
+              if (!preview) return null;
+              if ('toAsset' in preview) {
+                return previewSwap(preview as AssetSwapActionPreview, fromNetwork, fromChainTitle, 'PLR_DAO');
+              }
+            })}
+          </>
+        )}
+        <TransactionAction>
+          <ColoredText>Route</ColoredText>
+          <ValueWrapper>
+            <RoundedImage title={providerName ?? 'Unknown'} url={providerIconUrl} />
+            <ValueBlock>
+              <Text size={12} marginBottom={2} medium block>
+                {providerName}
+              </Text>
+              <Text size={16} medium>
+                {toAmount} {toAsset.symbol}{' '}
+              </Text>
+            </ValueBlock>
+          </ValueWrapper>
+          <ValueWrapper>
+            {!!cost && (
+              <>
+                <ColoredText>Gas price</ColoredText>
+                {cost}
+              </>
+            )}
+            {crossChainAction.transactions[crossChainAction.transactions.length - 1].createTimestamp && (
+              <>
```

Comment:
maybe create const for `crossChainAction.transactions[crossChainAction.transactions.length - 1].createTimestamp` just before render return